templates/observability-remote-write-proxy.yml

```diff
@@ -0,0 +1,335 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: observability-remote-write-proxy
+  annotations:
+    description: "Prometheus Remote Write Proxy for Observability"
+objects:
+  - kind: Deployment
+    apiVersion: apps/v1
+    metadata:
+      labels:
+        app.kubernetes.io/component: prometheus-remote-write-proxy
+        app.kubernetes.io/name: observability-remote-write-proxy
+        app.kubernetes.io/version: ${OBSERVABILITY_REMOTE_WRITE_PROXY_IMAGE_TAG}
+      name: observability-remote-write-proxy
+    spec:
+      replicas: ${{OBSERVABILITY_REMOTE_WRITE_PROXY_REPLICAS}}
+      strategy:
+        rollingUpdate:
```

Comment:
Progressive rollout of newer Observability Remote Write Proxy images on the Deployment